pkgs/applications/networking/remote/tlclient/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+
+, alsaLib
+, libX11
+, pcsclite
+}:
+let
+  tlclient-desktop = fetchurl {
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/tlclient.desktop?h=tlclient";
+      sha256 = "1219pqcbzh38ffxlz26pmn2yq6d2r3s0bjn6yaz319gnl9zl39ya";
+  };
+  tlclient-png = fetchurl {
+      url = "https://aur.archlinux.org/cgit/aur.git/tree/tlclient.png?h=tlclient";
+      sha256 = "0ppgkaampv8xlz1qvymbj8d26jhf9xg221jwlmc5zdssdjm631g4";
+  };
```

Comment:
Same here, pin to a commit instead: https://aur.archlinux.org/cgit/aur.git/plain/tlclient.png?h=tlclient&id=fd126edd54a012caa8c9ce47ab8d7fb25e982e74

Reply:
Fixed.